### PR TITLE
Add language switcher to member area

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Add language switcher to member area

- Add Language selector to member area menu. Change select size from default medium to small, looks better this way
- Add alternates head links to member area so that language switcher lands on the right page instead of falling back to website index page
- Rename `LanguageSwitcherForm` to `LanguageSelector`
- Move footer-specific styles for language selector to footer itself

Solution for providing alternates looks awkward, but it's more surgical than rewriting all member area to app router as a part of this PR

<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/GLY3tSM85RFBvAf6lFNA/cd4a0457-8438-4977-ad7d-df92b39b3145.mov">
              
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/GLY3tSM85RFBvAf6lFNA/cd4a0457-8438-4977-ad7d-df92b39b3145.mov">Screen Recording 2024-08-19 at 11.37.49.mov</video>

## Justify why they are needed

UX improvement for Pet comparison site flow. Also, makes generic member area experience better

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4618?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4618** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4618?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4617** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4617?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4616** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4616?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/blocks/FooterBlock/FooterBlock.css.ts [hunk 1]
```diff
@@ -10,6 +10,24 @@ export const wrapperStyle = style({
...
+
+  '@media': {
+    [minWidth.md]: {
+      gridRow: 2,
+      gridColumn: '7 / -1',
+    },
+    [minWidth.xxl]: {
+      gridColumn: '7 / span 4',
+    },
+  },
```
Comment (guilhermespopolin) [inline apps/store/src/blocks/FooterBlock/FooterBlock.css.ts:28]: Should we use `responsiveStyles` here?

**Review by guilhermespopolin — APPROVED** [review]
So we're adding alternate links for that page and then, when locale is changed, we do a hard refresh with the url present on an alternate link for the chosen locale? 🤔

File: apps/store/src/blocks/FooterBlock/FooterBlock.css.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/blocks/FooterBlock/FooterBlock.css.ts:28]: For now it's just moved without changes, but we should at some point

**Comment by alebedev** [comment]
@guilhermespopolin you're correct, this is how language switcher works - it looks for alternate in target language and does a hard navigation. Or goes to index page in target language as a fallback

**Comment by alebedev** [comment]
### Merge activity

* **Aug 19, 10:01 AM EDT**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4618).

**Pull request merged**